packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+import { IPressableState, IFocusable } from '@fluentui-react-native/interactive-hooks';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextProps } from '@fluentui-react-native/text';
+import { ITextTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
+
+export const checkboxName = 'Checkbox';
+
+export interface ICheckboxState extends IPressableState {
+  checked?: boolean;
+
+  disabled?: boolean;
+
+  // True if boxSide='end'
+  boxAtEnd?: boolean;
+}
+
+export interface ICheckboxProps extends IPressableProps {
+  ariaLabel?: string;
+
+  /*
+   ** Checked state. Mutually exclusive to “defaultChecked”. Use this if you control the checked state at a higher level
+   ** and plan to pass in the correct value based on handling onChange events and re-rendering.
+   */
+  checked?: boolean;
+
+  /*
+   ** Default checked state. Mutually exclusive to ‘checked’. Use this if you want an uncontrolled component, and
+   ** want the Checkbox instance to maintain its own state.
+   */
+  defaultChecked?: boolean;
+
+  boxSide?: 'start' | 'end';
+
+  disabled?: boolean;
+
+  label?: string;
+
+  /**
+   * A RefObject to access the IButton interface. Use this to access the public methods and properties of the component.
```

Comment:
nit IFocusable interface?

Reply:
Nice catch, changing in newest iteration